packages/editor-ui/src/views/canvasHelpers.ts

```diff
@@ -48,6 +48,20 @@ export const DEFAULT_START_NODE = {
 	parameters: {},
 };
 
+export const WELCOME_STICKY_NODE = {
+	name: "Note1",
```

Comment:
would give it more unique name... that way you could use this to distinguish it when implementing telemetry events for this feature